fem/bilininteg.hpp

```diff
@@ -1872,7 +1874,7 @@ class DiffusionIntegrator: public BilinearFormIntegrator
    const GeometricFactors *geom;  ///< Not owned
    int dim, ne, dofs1D, quad1D;
    Vector pa_data;
-
+   bool symmetric = true; ///< False if using a nonsymmetric matrix coefficient
```

Comment:
This flag is used only in the PA setup and multiplication, and I think it is clear if the user understands the kernels; otherwise, the user should ignore it. Maybe `pa_symmetric` would be a more appropriate name.